homework_2/internal/infrastructure/http/axilisSecondOfferFeed.go

```diff
@@ -0,0 +1,95 @@
+package http
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"code-cadets-2021/homework_2/internal/domain/models"
+)
+
+const secondFeedURL = "http://18.193.121.232/axilis-feed-2"
+
+type AxilisSecondOfferFeed struct {
+	updates    chan models.Odd
+	httpClient *http.Client
+}
+
+func NewAxilisSecondOfferFeed(
+	httpClient *http.Client,
+) *AxilisSecondOfferFeed {
+	return &AxilisSecondOfferFeed{
+		updates:    make(chan models.Odd),
+		httpClient: httpClient,
+	}
+}
+
+func (a *AxilisSecondOfferFeed) Start(ctx context.Context) error {
+	defer close(a.updates)
+	defer log.Printf("shutting down %s", a)
+
+	for {
+		select {
+		case <-ctx.Done():
+			return nil
+
+		case <-time.After(time.Second * 3):
+			response2, err := a.httpClient.Get(secondFeedURL)
```

Comment:
within this context, I don't think there is any need to call this `response2` as there is no `response` or `response1` anywhere in this file